hugo.toml

```diff
@@ -305,6 +305,18 @@ i18nDir = "./data/i18n"
 [languages.en.params]
 description = "Production-Grade Container Orchestration"
 
+[languages.bn]
+title = "Kubernetes"
+weight = 15
```

Comment:
Value for `weight` configuration may require fine-tuning to ensure proper sorting order for all languages.